src/components/CreateComment.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {useState} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { 
+    Container, 
+    TextField, 
+    Grid, 
+    Button
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+    root: {
+        marginTop: '2rem',
+        marginBottom: '2rem',
+        borderStyle: 'solid'
+    },
+
+    author: {
+        width: '100%',
+        marginTop: '1rem',
+        marginBottom: '1rem',
+    },
+
+    actionButton: {
+        marginTop: '2rem',
+        marginBottom: '2rem',
+        display: 'flex',
+        justifyContent: 'space-around',
+        alignItems: 'center'
+    },
+
+    submitButton: {
+        padding: '0.75rem',
+        paddingLeft: '7rem',
+        paddingRight: '7rem',
+        background: 'lightGreen'
+    }, 
+});
+
+
+const CreateComment: React.FC = () => {
+    const classes = useStyles();
+
+    const [comment, setComment] = useState(
+        {
+            author: '',
+            content: ''
+        }
+    );
+
+    return(
+        <Container className={classes.root}>
+            <Grid container>
+                <Grid item xs={12}>
+                    <TextField                         
+                        id="content"
+                        margin="normal"
+                        fullWidth
+                        multiline
+                        rows={8}
+                        rowsMax={50}
+                        label="Comment"
+                        variant="outlined"
+                        value={comment.content}
+                        onChange={e => {
```

Comment:
hrmm, I wonder how much of this could be abstracted in one function

Reply:
I'm all for abstraction, but again we are dealing with beginners to coding. My goal was to make as less confusion as possible

Reply:
That's fair, I'm fine with keeping these inline for that reason, it's not uncommon or a bad practice